client/src/constants.js

```diff
@@ -139,3 +139,34 @@ export const countryData = [
     ]
   }
 ]
+export const quarterInfo = [
```

Comment:
why do we need this? Can you think of other logic? anything at server side?

Reply:
Done at server side